docs/modules/ROOT/pages/Scala_Build_Examples.adoc

```diff
@@ -48,21 +48,18 @@ include::example/scalabuilds/7-cross-scala-version.adoc[]
 
 == SBT-Compatible Modules
 
-include::example/scalabuilds/8-sbt-compat-modules.adoc[]
+include::example/scalabuilds/8-compat-modules.adoc[]
 
-== Java Modules
-
-include::example/scalabuilds/9-java-modules.adoc[]
 
 == Realistic Scala Example Project
 
-include::example/scalabuilds/10-scala-realistic.adoc[]
+include::example/scalabuilds/9-realistic.adoc[]
 
 
-== Example Builds
+== Real World Builds
```

Comment:
Maybe, we should add links to some real projects, which are known to be build with Mill.

Reply:
Possible, but for now that would only include Scala projects, since there aren't any well known Java projects using Mill (yet)

Reply:
I think c3p0 is using Mill now, but it's a rather bespoken build.

Reply:
Oh that's true! I added c3p0 as an example in the Java page and Ammonite/ScalaCLI/Coursier as examples in the Scala page